model.py

```diff
@@ -32,24 +32,28 @@ class Shortlist:
 #functions
 
 
-def load_pickle(path):
-    with open(path/"shortlist.pickle", "rb") as f:
+def load_pickle(path,filename="shortlist.pickle"):
```

Comment:
I wonder if this is what is meant when you want the file name to be a variable?

Reply:
It doesn't need to be an argument in the function. In `model.py`, before the functions, create a constant:

```
PICKLE_FILENAME = "shortlist.pickle"
```

Then in the functions, use this variable name rather than the hardcoded string.

Reply:
Is it because this way it's easier to change the name of the pickled file?

Reply:
Yes, exactly.